code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -1160,16 +1160,25 @@ GLOBAL_LIST_EMPTY(features_by_species)
 				return FALSE
 		user.do_attack_animation(target, atk_effect)
 
+		var/off_balance = FALSE //has our target been shoved recently? If so, they're off-balance and we get an easy hit.
+		var/grappled = FALSE //Someone in a grapple is much more vulnerable to being harmed by punches.
+		if(target.has_movespeed_modifier(/datum/movespeed_modifier/shove))
+			off_balance = TRUE
+
+		if(target.pulledby && target.pulledby.grab_state >= GRAB_AGGRESSIVE)
+			grappled = TRUE
```

Comment:
```suggestion
		//has our target been shoved recently? If so, they're off-balance and we get an easy hit.
		var/off_balance = target.has_movespeed_modifier(/datum/movespeed_modifier/shove)
		//Someone in a grapple is much more vulnerable to being harmed by punches.
		var/grappled = target.pulledby && target.pulledby.grab_state >= GRAB_AGGRESSIVE
```
Not sure this is made more readable by being split across two lines?

Reply:
Funnily enough I thought I already did this. My bad, I'll clean this up.